core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -173,15 +173,6 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
     this
   }
 
-  /**
-   * Set multiple parameters together
-   */
-  @deprecated("Use setAll(Iterable) instead", "3.0.0")
-  def setAll(settings: Traversable[(String, String)]): SparkConf = {
-    settings.foreach { case (k, v) => set(k, v) }
-    this
-  }
-
```

Comment:
I'm okay with this, but this might be not aligned with the latest community decision at `Cost of Breaking an API`.
cc @gatorsmile

Reply:
Yeah, tough call here. The method call is still source compatible in virtually all cases. It'll be binary compatible in most cases too because callers will bind to the more specific Iterable overload anyway if passing almost any normal collection. This is why I deprecated this method in 3.0.0. Still it's a tough call removing it, but I do not see any way around it given the changes in Scala 2.13 (well we can fork source trees, but that seems like a huge pain).